spec/requests/edit_collection_spec.rb

```diff
@@ -108,6 +114,57 @@
             expect(response).to have_http_status(:found)
             expect(response).to redirect_to(collection)
           end
+
+          it 'sends emails to the managers' do
+            expect do
+              patch "/collections/#{collection.id}",
+                    params: { collection: collection_params, commit: save_draft_button }
+            end.to have_enqueued_job(ActionMailer::MailDeliveryJob).with(
+              'CollectionsMailer', 'participants_changed_email', 'deliver_now',
+              { params: { user: user, collection: collection }, args: [] }
+            )
+          end
+
+          it 'sends emails to the reviewers' do
+            expect do
+              patch "/collections/#{collection.id}",
+                    params: { collection: collection_params, commit: save_draft_button }
+            end.to have_enqueued_job(ActionMailer::MailDeliveryJob).with(
+              'CollectionsMailer', 'participants_changed_email', 'deliver_now',
+              { params: { user: collection.reviewers.first, collection: collection }, args: [] }
+            )
+          end
+
+          it 'logs the changes in the event description' do
+            patch "/collections/#{collection.id}",
+                  params: { collection: collection_params, commit: save_draft_button }
+            event_description = collection.reload.events.order(created_at: :desc).take.description
+            expect(event_description).to match(/Removed reviewers: w.a.sterner/)
+            expect(event_description).to match(/Removed depositors: #{removed_depositor.sunetid}/)
+          end
+
+          # rubocop:disable RSpec/NestedGroups
```

Comment:
This seems like a sane number of nested groups, you could bump up this threshold in .rubocop.yml